doc/overview.md

```diff
@@ -0,0 +1,52 @@
+
+## Record Keeping
+
+Saskatoon keeps several groups of records (*NOTE: this is only conceptual, not the actual database implementation*):
+
+- **personnel**
+    - *__tree owners__*: who offer fruit trees for harvest. Trees can be owned by an organization, e.g. McGill University.
+    - *__pick-leaders__*: who lead individual harvests from start to end, i.e. organize and oversee picks, transport equipment to and from harvests, bring fruit to beneficiary organizations and log the day’s bounty in Saskatoon.
+    - *__volunteers / harvesters__*: who help pick the fruits during a harvest. Record-keeping of harvesters can help pick-leaders assemble the harvest teams, e.g. offering more opportunities to first-time harvesters when the participation requests exceed the available spots.
+    - *__core members / admins__*: who have full access to the system and records, and can grant/revoke access and permission.
+- **properties / locations & fruit trees**: where and what to harvest. One property may have multiple types of fruit trees, and one type of fruit trees may have multiple harvests each year.
+- **beneficiary organizations**: external recipients of fruits. A portion of the harvest can be donated to local communities.
+- **harvests & participation requests**: Each harvest record contains various info, e.g. date & time, location, yield of each type of fruits, participants, how the fruits were distributed, and etc.
+- **equipments**: tools that pick leaders can borrow for harvests, e.g. ladders, bikes, scales, and etc.
+
+## Facilitating harvests
+
+### Pick-leader
+
+Pick-leaders use the Saskatoon dashboard to organize harvests and log the results afterward. A series of activities would happen around the harvest record during a harvest.
+
+| Stage | Status in harvest record | Time range in harvest record | Other info added in harvest record |
+| --- | --- | --- | --- |
+|1. A new harvest record for the current year is created in Saskatoon after the tree owner confirms the interest to participate this year. It usually happens before the season starts. | _Orphan_ | usually a 2-week period | property and trees |
+|2. A pick-leader declares to lead this harvest | _Adopted_ | same as previous | pick-leader |
+|3. The pick-leader confirms with the tree owner about the exact harvest date and time, contacts beneficiary organizations to receive fruits for this harvest, and publish the event on harvest calendar to call for participants | _Date scheduled_ | A specific date and time | beneficiary organizations, participation requests |
+| 4. The pick-leader assembles the team for harvest by accepting/declining participation requests | _Ready_ | same as previous | participation requests status updated |
+| 5. Harvest ends. If successful, the pick-leader logs the harvest yield and how the fruits were distributed among the tree owner, volunteers, and beneficiary organizations | _Successful / Cancelled_ | same as previous | harvest yield and distribution info |
```

Comment:
can you please format the table so it's easier to read in plain text

Reply:
It just occurred to me that maybe we can enable the wiki space in the repo and put these high-level docs there? The wiki pages supports tables and diagrams better, and it would be simpler to update (i.e. not branch merging involved). What do you think?

Reply:
We can absolutely move the docs to the Wiki but I would like to keep the same PR review process
https://docs.github.com/en/communities/documenting-your-project-with-wikis/adding-or-editing-wiki-pages#adding-or-editing-wiki-pages-locally

Reply:
The table is formatted in the new commit.

Thanks for the info on GitHub wiki! I did a bit research and experiment, and found that despite allowing basic git actions on the wiki repo (e.g. clone, commit, branch), GitHub doesn't really support normal processes on the web interface (e.g. view branches, create pull request). You may do it with some hacks, but that would be a pain. https://stackoverflow.com/questions/10642928/how-can-i-make-a-pull-request-for-a-wiki-page-on-github/56810747#56810747

Thus I think for now it is easier to have the docs still reside in /doc folder in the main repo.

Reply:
> Thus I think for now it is easier to have the docs still reside in /doc folder in the main repo.
I invited you as a collaborator with write access so you can push to the wiki. I'm also fine with keeping .md files in the `/doc` folder and we can migrate at a later point. Whichever is easier for you.

> Shall these new docs & doc changes go to the master branch instead?
Yes sorry for that, we should update the contributing docs, we no longer want PRs to target the develop branch